irods/manager/user_manager.py

```diff
@@ -27,6 +28,25 @@ def get(self, user_name, user_zone=""):
             raise UserDoesNotExist()
         return iRODSUser(self, result)
 
+    # pending resolution of https://github.com/irods/irods/issues/6887
+    def create_with_password(self, user_name, password, user_zone = ''):
+        """Typically used by a groupadmin to initialize the password field while creating the new user.
+           (This is necessary since group administrators may not change the password of an existing user.)
+        """
```

Comment:
Now that I review it maybe the word 'typically' is wrong, it does imply an older API would have had it.   No, it's a new function, and I don't know if there'd be a better way to introduce it, or name it.  Or if I perhaps  just need to change the comment to say:  "This  method  can be used by a groupadmin to ..."

Reply:
The "typically" was throwing me off. I think changing the statement so that it is clear in who can use it would help a great deal.

Everything looks fine.

Reply:
ok... changed the docstring and also removed the comment regarding resolution of irods/irods#6887 , as that issue is closed.